tools/genwqe_memcopy.c

```diff
@@ -875,11 +875,11 @@ int main(int argc, char *argv[])
 		if (total_usec) {
 			if (total_usec < 100000) {
 				kibs = ((bytes_copied * 1000000) / 1024) / total_usec;
-				VERBOSE0("%d KiB, in %ld usec, %ld KiB/sec", kib, total_usec, kibs);
+				VERBOSE0("%d KiB, in %lld usec, %lu KiB/sec", kib, total_usec, kibs);
```

Comment:
VERBOSE0("%d KiB, in %llu usec, %lu KiB/sec", kib, (long long)total_usec, kibs);